src/lib/api/getPaginatedJobsBySearch.js

```diff
@@ -1,7 +0,0 @@
-import axios from 'axios'
```

Comment:
`getPaginatedJobsBySearch` is using pagination on the backend so we don't load all of the documents in our database to the client at once. I think it may be worth it to implement the API calls using this call and deprecate the above `getJobs` and `getJobsBySearch`, since they're functionality is encompassed by this method.